Library/Homebrew/test/Gemfile

```diff
@@ -3,5 +3,17 @@ source "https://rubygems.org"
 gem "mocha", "~> 1.1"
 gem "minitest", "~> 5.3"
 gem "rake", "~> 10.3"
-gem "simplecov", "~> 0.10.0", :require => false
-gem "coveralls", "0.8.9", :require => false
+# This is a patched version of the v0.11.1. Switch back to the stable version
+# when #436 and #437 have been merged upstream.
+# https://github.com/colszowka/simplecov/pull/436
```

Comment:
0.11.2 fixed https://github.com/colszowka/simplecov/pull/436 but not https://github.com/colszowka/simplecov/pull/437, for which we need to wait for 0.11.3. I updated our fork as well as this PR to use a patched 0.11.2 and I’m writing tests in the meantime to reduce the coverage drop.